R/testdata.R

```diff
@@ -590,6 +590,27 @@ dataset_test_worker <-
             process_format_contexts(dataset_id, data)
         )
 
+        # Check that there are no duplicate `var_in` or `context_property` fields
+        context_properties <- sapply(metadata[["contexts"]], "[[", "context_property")
+        context_vars_in <- sapply(metadata[["contexts"]], "[[", "var_in")
+
+        expect_equal(
+          context_properties |> duplicated() |> sum(),
```

Comment:
I feel like Daniel at some point asked that we just use `%>%` to standardise but I might be wrong

Reply:
Either is fine for now. At some point we might unify, but not everything is perfectly interchangeable so can't just find replace.